README.md

```diff
@@ -594,6 +594,16 @@ This is the **only** place in Puppet where backslash separators should be used.
 Note that backslashes in double-quoted strings *must* be double-escaped and
 backslashes in single-quoted strings *may* be double-escaped.
 
+> **Note on Secrets in `install_options`
```

Comment:
This shouldn't be a markdown quote. `Note` should have ** before and after

Reply:
Changed to `**WARNING** about...` and dropped block quotes.